spec/lib/http_client_spec.rb

```diff
@@ -4,6 +4,7 @@
 
 describe HTTPClient do
   let(:client) { HTTPClient.new }
+  before { HttpLog.configure { |c| c.logger = Logger.new @log } }
```

Comment:
Why is this necessary here, isn't that the default already?

Reply:
I removed this default behavior from spec_helper.rb to be able to set another logger in tests

Reply:
Yep, I get that, but this is the default which is now set in `spec/lib/http_log_spec.rb` - I don't understand the need to set it explicitly again here, just for HttpClient - does it actually change anything from the default?

Reply:
I did that because HTTPClient test had failed :) It looks like spec/lib/http_log_spec.rb and spec/lib/http_client_spec.rb are independent files. And before actions from one file are not applied to another.